### PR TITLE
Move filter classes into separate file

### PR DESCRIPTION
**Pull request by mberz**
Separate the filter objects form the coefficient calculating generating functions.

### CODE REVIEW

File: tests/test_filter.py [hunk 1]
```diff
@@ -2,7 +2,7 @@
 from unittest import mock
 import numpy as np
 import numpy.testing as npt
-import pyfar.dsp.filter as hpfilter
+import pyfar.dsp.classes as hpfilter
```
Comment (f-brinkmann) [inline tests/test_filter.py:5]: hpfilter sounds like haiopy - how about pfilt or something like that?

**Review by f-brinkmann — APPROVED** [review]
Only one naming suggestion - ready to merge from my point of view...

File: tests/test_filter.py (hunk 1 above)
Comment (mberz) [reply tests/test_filter.py:5]: true, it's only in the tests though.

**Comment by mberz** [comment]
> Only one naming suggestion - ready to merge from my point of view...

👍

**Pull request merged**